package.json

```diff
@@ -19,17 +19,21 @@
     "npm": ">=3.0.0"
   },
   "scripts": {
-    "lint": "aegir-lint",
-    "coverage": "gulp coverage",
-    "test": "gulp test --dom",
-    "test:node": "npm run test:unit:node",
-    "test:browser": "npm run test:unit:browser",
-    "test:unit:node": "gulp test:node",
-    "test:unit:node:core": "TEST=core npm run test:unit:node",
-    "test:unit:node:http": "TEST=http npm run test:unit:node",
-    "test:unit:node:gateway": "TEST=gateway npm run test:unit:node",
-    "test:unit:node:cli": "TEST=cli npm run test:unit:node",
-    "test:unit:browser": "gulp test:browser",
+    "lint": "aegir lint",
+    "build": "aegir build",
+    "test": "aegir test --target node --target browser --no-cors --no-parallel",
+    "test:node": "aegir test --target node --no-parallel",
+    "test:browser": "aegir test --target browser --no-parallel",
+    "release": "aegir test release --target node --target browser --no-parallel",
+    "release-minor": "aegir release --type minor --target node --target browser --no-parallel",
+    "release-major": "aegir release --type major --target node --target browser --no-parallel",
+    "coverage": "aegir coverage",
+    "test:unit:node": "aegir test --target node --no-parallel",
+    "test:unit:node:core": "TEST=core aegir test --target node --no-parallel",
+    "test:unit:node:http": "TEST=http aegir test --target node --no-parallel  ",
+    "test:unit:node:gateway": "TEST=gateway aegir test --target node --no-parallel",
+    "test:unit:node:cli": "TEST=cli aegir test --target node --no-parallel",
```

Comment:
With the new AEgir, you should be able to do something better than setting an env var. See https://github.com/ipfs/aegir/issues/135

Reply:
@dryajov are you on it?